app/scripts/controllers/transactions/index.js

```diff
@@ -1474,24 +1479,12 @@ export default class TransactionController extends EventEmitter {
     const signedEthTx = await this.signEthTx(
       unsignedEthTx,
       fromAddress,
-      ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-      txMeta.custodyStatus ? txMeta : undefined,
-      ///: END:ONLY_INCLUDE_IN
+      this._getAdditionalSignArguments?.(txMeta),
     );
 
-    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-    if (txMeta.custodyStatus) {
-      txMeta.custodyId = signedEthTx.custodian_transactionId;
-      txMeta.custodyStatus = signedEthTx.transactionStatus;
-
-      this.transactionUpdateController.addTransactionToWatchList(
-        txMeta.custodyId,
-        fromAddress,
-      );
-
+    if (this._afterSign?.(txMeta, signedEthTx)) {
```

Comment:
Would it be clearer conceptually for a return value of `false` to indicate that signing shouldn't take place as `true` implies positive which means also can continue as normal?

Reply:
indeed, makes total sense. I pushed this change.